packages/atomic-layout/test/ssr.spec.tsx

```diff
@@ -25,4 +26,13 @@ describe('Server-side rendering', () => {
 
     expect(renderOnServer).not.toThrow()
   })
+
+  it('renders with the src attribute', () => {
+    const StyledImage = styled.img`
+      border-radius: 5px;
+    `
+    const ElementImage = useResponsiveComponent(StyledImage)
+    const ElementImageHtml = renderToString(<ElementImage src="foo.png" />)
+    expect(ElementImageHtml).toContain('src="foo.png"')
```

Comment:
Now I understand why you used substring match. Rendering a component to a string leaves us with string assertions. I wonder, in which environment does `react-testing-library` render? I'd suggest to research that, and if we can make it render in a server-like environment we should use it.

Reply:
@kettanaito I couldn't find any documentation for SSR context from `react-testing-library`, I opened an issue for it already, maybe there is a way I'm not sure as of now.
this is the issue: https://github.com/testing-library/react-testing-library/issues/561